__tests__/app.test.js

```diff
@@ -12,29 +12,45 @@ beforeEach(() => {
 
   describe('GET /api/topics', () => {
       describe('Functionality', () => {
-        test('Returns an array of topic objects with slug and decription properties', () => {
-        return request(app)
+        test('Returns an array of topic objects with slug and decription properties', async () => {
+        const response = await request(app)
```

Comment:
Believe it or not, it's a more widely accepted convention here to use `return` instead of `const foo = await...`